docs/modules/ROOT/pages/spring-cloud-netflix.adoc

```diff
@@ -267,25 +267,31 @@ Do not use the `EurekaClient` in a `@PostConstruct` method or in a `@Scheduled`
 It is initialized in a `SmartLifecycle` (with `phase=0`), so the earliest you can rely on it being available is in another `SmartLifecycle` with a higher phase.
 ====
 
-==== EurekaClient with Jersey
+==== Underlying HTTP clients
+
+`EurekaClient` uses either `RestTemplate`, `WebClient` or `JerseyClient` under the hood. In order to use the `EurekaClient`, you need to have one of the supported HTTP clients on your classpath.
```

Comment:
Change to:

`EurekaClient` uses one of `RestTemplate`, `WebClient`, or `JerseyClient` under the hood. To use the `EurekaClient`, you need to have one of the supported HTTP clients on your classpath.

That usage of "either" isn't, strictly speaking, wrong, but it is about a century out of date. Also, as a general rule, you rarely need "In order" - you can often leave it out and have the same meaning, as here.